CONTRIBUTING.md

```diff
@@ -25,3 +25,32 @@ Once ready for review, make sure to include a thorough PR description to help re
 We use the `git rebase` command to keep our commit history tidy.
 Rebasing is an easy way to make sure that each PR includes a series of clean commits with descriptive commit messages
 See [this tutorial](https://docs.gitlab.com/ee/topics/git/git_rebase.html) for a detailed explanation of `git rebase` and how you should use it to maintain a clean commit history.
+
+### Versioning
+
+When we need to fix bugs or introduce new features in our npm packages, we update the package version. To ensure consistency and simplicity in this process, we use a tool called [changesets](https://github.com/changesets/changesets).
+
+Each changeset contains three key pieces of information:
+* The package(s) to be published
+* Whether the release is major, minor, or a patch
+* A description of the change, which will be added to the autogenerated CHANGELOG.md file for the package.
+
+To create a changeset, simply run `pnpm changeset`. This command will guide you through the process of selecting packages and defining the changes. Once complete, a new file will be generated, which you need to commit to the repository. Here's an [example changeset](https://github.com/ethereum-optimism/ecosystem/blob/500f8ec30fbeda4bd675457f68e3f877f2533a03/.changeset/purple-rocks-destroy.md?plain=1).
```

Comment:
I'd be clear on when in the process you create a changeset. "When you are ready to publish changes to a package, you should run this command" or something like that. Might feel self-explanatory but would be good to call it out.